### DISCUSSION TITLE
Creating SSD with windows

### DISCUSSION THREAD

**Discussion by controlfix87** [osy86/HaC-Mini#911] [Q&A] [answered]
Hi, 
How can I create another SSD that will be in the second slot of the NVMe to boot windows? 

Thanks :)

**Comment by scottzoloft** [comment]
closest way i can think of is to unplug your mac SSD and install windows normally on its own SSD without doing bootcamp. plug both in and use your bios boot selector to select your OS. might have to mash on the option (alt?) key on your keyboard when selecting opencore to prevent it from automatically booting to windows

i haven't tried this and i'm just guessing that's how it would work since bootcamp will only let you partition your main drive to install windows.

**Reply by controlfix87** [reply]
Thanks. But I want to do this on a separate ssd and I don’t want windows to install Boot Manager on it so it won’t conflict with OpenCore. 
How about using WinToUsb? Will this do the trick? 
If not, maybe I can detach the Mac SSD and install windows on the new SSD and then return the Mac SSD.. 
any thoughts? 

Thanks

**Reply by controlfix87** [reply]
> closest way i can think of is to unplug your mac SSD and install windows normally on its own SSD without doing bootcamp. plug both in and use your bios boot selector to select your OS. might have to mash on the option (alt?) key on your keyboard when selecting opencore to prevent it from automatically booting to windows
> 
> i haven't tried this and i'm just guessing that's how it would work since bootcamp will only let you partition your main drive to install windows.

I did this finally, changed BIOS to boot from OpenCore. 
The system is recognized as macmini8,2 or something like that.. how can I avoid it? 
Change BIOS to boot from windows boot manager?

**Reply by scottzoloft** [reply]
> > closest way i can think of is to unplug your mac SSD and install windows normally on its own SSD without doing bootcamp. plug both in and use your bios boot selector to select your OS. might have to mash on the option (alt?) key on your keyboard when selecting opencore to prevent it from automatically booting to windows
> > i haven't tried this and i'm just guessing that's how it would work since bootcamp will only let you partition your main drive to install windows.
> 
> I did this finally, changed BIOS to boot from OpenCore.
> The system is recognized as macmini8,2 or something like that.. how can I avoid it?
> Change BIOS to boot from windows boot manager?

If you boot into windows from opencore windows will pull the mac system info from your EFI partition. If you select windows boot manager or the drive windows is installed on from BIOS windows should pull normal system info.

Idk enough about efi bootloading but I'd imagine if you don't want to f10 every time you want to select an OS, maybe you could chain bootloaders with grub or something? It might be possible to config grub to look at the opencore and windows EFIs, and then have BIOS default to boot to grub. Someone reading may know better

**Comment by controlfix87** [comment, marked as answer]
1. Detached the Mac SSD.
2. Install Windows, drivers, etc
3. Put back the Mac SSD
4. F10 boot menu select 'Windows Boot Manager', Reads system info normaly from EFI/Microsoft
